modules/end-user-guide/pages/defining-a-common-ide.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: PROCEDURE
+:navtitle : Defining a common IDE
+:description: Defining a common IDE
+:keywords: select-IDE, che-editor, che-editor-yaml
+:page-aliases: specifying-an-in-browser-ide-for-a-git-repository-by-using-che-editor.yaml.adoc, selecting-an-in-browser-ide-for-all-workspaces-that-clone-the-same-git-repository.adoc
+
+[id="defining-a-common-ide"]
+= Defining a common IDE
+
+While the xref:url-parameter-for-the-ide.adoc[URL parameter for the IDE] enables you to start a workspace with your personal choice of the supported IDE, you might find it more convenient to define the same IDE, even with a more detailed IDE configuration, for all workspaces that will clone the same remote Git repository with your project source code. To do so, use the `che-editor.yaml` file.
+
+TIP: If you intend to start most or all of your organization's workspaces with the same IDE other than Microsoft Visual Studio Code - Open Source, an alternative is for the administrator of your organization's {prod-short} instance to specify another supported IDE as the default IDE at the {prod-short} instance level. This can be done with `.spec.devEnvironments.defaultEditor` in the `CheCluster` Custom Resource.
```

Comment:
Very long, very complex. Can you make the tip more minimalistic, please?

Reply:
:) Yes, I know. I wish I could, but this is I'm afraid the shortest I can do because this is a complex subject.